proxy/src/telemetry/metrics/mod.rs

```diff
@@ -27,7 +26,9 @@ use control::pb::proxy::telemetry::{
     TransportSummary,
 };
 use ctx;
-use telemetry::event::Event;
+use telemetry::event::{Event};
```

Comment:
Is this the correct style for a single import?

Reply:
Did not mean to do that!